packages/api-service/tracks.ts

```diff
@@ -0,0 +1,51 @@
+import { APIResponse, ClientGetter, Cover, CoverMetadata, PagedRequest, PagedResponse } from './types'
+
+export interface Track {
+  id: number
+  creator_id: number
+  title: string
+  duration: number
+  album: string | null
+  artist: string
+  cover: string
+  cover_metadata: CoverMetadata
+  images: {
+    small: Cover
+    medium: Cover
+  }
+  status: 'Paid'
```

Comment:
I suspect this is an enum, but I'm guessing it can also contain `Free` perhaps? Either way, can be added/changed later if it's not certain.